packages/server/src/api/students.rs

```diff
@@ -2,7 +2,7 @@ use std::{collections::HashMap, str::FromStr};
 
 use actix_web::{
     get, post, put,
-    web::{Data, Json, Query},
+    web::{Data, Json, Path, Query},
     HttpMessage, HttpRequest, HttpResponse,
 };
 
```

Comment:
we need one get API call for catalog:

GET catalog
no 'get' prefix.

if we dont have query param - get all.
if we have, sort.

Reply:
query param:
GET ..../sogrim/api/students/catalog/?faculty=nisso